src/models/WidgetData.ts

```diff
@@ -156,3 +156,11 @@ export interface IWidgetAccidentCountByCarType extends IWidgetDataBase {
     percentage_country: number;
   }[];
 }
+export interface IWidgetInjuredAccidentsWithPedestrians extends IWidgetDataBase {
+  items: {
+    YEAR: number;
+    KILLED_INJURY_SEVERITY_COUNT: string | number;
```

Comment:
pass data as is - doe conversions inside the component if required

Reply:
done